fx_crash_sig/tests.py

```diff
@@ -24,7 +28,7 @@ def test_symbolicate_single2(self):
 
     def test_symbolicate_none(self):
         symbolicated = self.symbolicator.symbolicate(None)
-        self.assertEquals(symbolicated, {})
+        self.assertEqual(symbolicated, {})
```

Comment:
`assertEquals` is deprecated in favor of `assertEqual` (with no s) so I fixed that.